src/Frame.cpp

```diff
@@ -90,32 +119,13 @@ void Frame::PostUnserializeFixup(Frame *f, Space *space)
 		PostUnserializeFixup(kid, space);
 }
 
-void Frame::Init(Frame *parent, const char *label, unsigned int flags)
-{
-	m_sfx = 0;
-	m_sbody = 0;
-	m_astroBody = 0;
-	m_parent = parent;
-	m_flags = flags;
-	m_radius = 0;
-	m_pos = vector3d(0.0);
-	m_vel = vector3d(0.0);
-	m_angSpeed = 0.0;
-	m_orient = matrix3x3d::Identity();
-	m_initialOrient = matrix3x3d::Identity();
-	ClearMovement();
-	m_collisionSpace = new CollisionSpace();
-	if (m_parent) m_parent->AddChild(this);
-	if (label) m_label = label;
-	m_sbody = nullptr;
-}
-
 Frame::~Frame()
 {
-	m_sfx.reset();
```

Comment:
So what handles this now?

Reply:
...dtor it self, if I'm not wrong: it's a unique_ptr, thus it should call dtor of pointed object automatically